modules/ocf_printhost/files/enforcer-pc

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+gs -o /dev/null -sDEVICE=bbox $1 2>&1 | grep HiResBoundingBox | wc -l
```

Comment:
It'd be cool to have tests for this file (e.g. a bunch of postscript files we cobbled together along with expected page count).

That would give us a lot more confidence when changing things in the future.

(but maybe a long-term goal)